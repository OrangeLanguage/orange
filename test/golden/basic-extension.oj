extern Int
def Int.inc() this.add(1)