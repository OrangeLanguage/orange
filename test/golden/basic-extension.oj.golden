(() => {
  _Int.prototype.inc = (function (_cont) {
    _cont = _cont();
    return (this.add)(() => new _Int(1n), () => (function (_0) {
      _0 = _0();
      return _cont(() => _0);
    }));
  });
  extern;
  Int;
  return _Int.prototype.inc;
})()